### PR TITLE
feat: Generate new standard non-security compose file with UI included

### PR DESCRIPTION
**Pull request by lenny-intel**
Removed all stand-alone UI files, commands and options

closes

Signed-off-by: lenny <leonard.goodell@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
UI is a stand-alone compose file that must be run separately after running the main compose file. Both must be run with the `-p edgex` option so the UI compose can find the edgex network.

## What is the new behavior?
New standard non-security compose file with UI included is now generated. This compose is only one that needs to be started.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/README.md [hunk 1]
```diff
@@ -121,25 +119,25 @@ portainer-down	Stops Portainer independent of the EdgeX services
 
 ```
 build
-Generates the all standard Edgex compose file variations and the TAF testing compose files. The generated compose files are stored in the top level folder. Each variation or standard compose files, except UI, includes Device REST & Device Virtual. Compose files are named appropriately for options used to generate them. TAF compose files are store in the 'taf' sub-folder
+Generates the all standard Edgex compose file variations and the TAF testing compose files. The generated compose files are stored in the top level folder. Each variation or standard compose files, ex includes Device REST & Device Virtual. Compose files are named appropriately for options used to generate them. TAF compose files are store in the 'taf' sub-folder
```
[the remark this answers is not in the document]
Comment (lenny-intel) [reply compose-builder/README.md]: Yup, but needs more rewording.

[the remark this answers is not in the document]
Comment (lenny-intel) [reply compose-builder/README.md]: Done

**Review by jim-wang-intel — APPROVED** [review]

**Comment by cherrycl** [comment]
recheck

**Review by badboy-huaqiao — APPROVED** [review]
LGTM

**Pull request merged**